Makefile

```diff
@@ -284,7 +289,8 @@ out/doc/api/%.json: doc/api/%.md
 	[ -x $(NODE) ] && $(NODE) $(gen-json) || node $(gen-json)
 
 # check if ./node is actually set, else use user pre-installed binary
-gen-html = tools/doc/generate.js --node-version=$(FULLVERSION) --format=html --template=doc/template.html $< > $@
+gen-html = tools/doc/generate.js --node-version=$(FULLVERSION) --format=html \
+			--template=doc/template.html --analytics=$(DOCS_ANALYTICS) $< > $@
```

Comment:
@Fishrock123 that's correct, because of [tools/doc/html.js#130](https://github.com/nodejs/node/pull/6601/files#diff-de394d26dd6b5ebee9f95daaeaf7e5e1R130).